spec/OBD/OBD.vspec

```diff
@@ -74,49 +85,39 @@
   description: PID 05 - Coolant temperature
 
 - ShortTermFuelTrim1:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
-  min: -100
-  max: 100
```

Comment:
We previously only had limits in a few places, and they were not always correct. Like here where the max limit actually is 99.21875 due to the scaling used.